ggshield/core/scan/scan_context.py

```diff
@@ -34,6 +37,10 @@ def get_http_headers(self) -> Dict[str, str]:
             "OS-Version": self.os_version,
             "Python-Version": self.python_version,
         }
+        if self.target_path is not None:
+            repo_url = get_repository_url_from_path(self.target_path)
+            if repo_url is not None:
+                headers["Repository-URL"] = repo_url
```

Comment:
According to the PR description, the header should be `GGShield-Repository-URL`.

Reply:
"Repository-URL" is added to `headers`, which are all transformed by adding the `GGShield-` prefix, in the return below, so it ends up as `GGShield-Repository-URL` as expected

Reply:
Ah right, I forgot about this 👍🏻.